src/client.js

```diff
@@ -320,6 +323,17 @@ export class Saturn {
     }
 
     if (lastError) {
+      const originUrl = opts.originURL ?? this.opts.originURL
+      // Use customer origin if cid is not retrievable by lassie.
+      if (originUrl) {
+        opts.nodes = Array({ url: originUrl })
+        try {
+          yield * fetchContent({ flatFile: true })
```

Comment:
We should modify the `format: car` param on line 173 instead of adding a new config. 

* Line 331: Pass `{ format: null }` as the option
* Line 413: Only add `format` param if `opts.format` is truthy.